pkg/controller/remotemachineset/gcpactuator.go

```diff
@@ -17,25 +25,32 @@ import (
 	"github.com/openshift/hive/pkg/gcpclient"
 )
 
+const (
+	// Omit m, the installer used this for the master machines.
+	validLeaseChars = "abcdefghijklnopqrstuvwxyz0123456789"
+)
+
 // GCPActuator encapsulates the pieces necessary to be able to generate
 // a list of MachineSets to sync to the remote cluster.
 type GCPActuator struct {
-	client gcpclient.Client
-	logger log.FieldLogger
+	client    client.Client
+	gcpClient gcpclient.Client
+	logger    log.FieldLogger
 }
 
 var _ Actuator = &GCPActuator{}
 
 // NewGCPActuator is the constructor for building a GCPActuator
-func NewGCPActuator(gcpCreds *corev1.Secret, logger log.FieldLogger) (*GCPActuator, error) {
+func NewGCPActuator(client client.Client, gcpCreds *corev1.Secret, logger log.FieldLogger) (*GCPActuator, error) {
```

Comment:
Did you intend to store `client` in the created actuator?

Reply:
I did yes. Is there a reason not to? Viewing it as an extension of the actuator and it needs several client calls now, seemed more logical than passing it around several times.

Reply:
No. What I meant was that you are not setting the client when I think that you meant to.

Reply:
LOL that's a good one, thanks. Would have hit that fast in real world  testing.